### ISSUE TITLE
Unable to visualise image with no Content-Type header (Version 7.3.2)

### ISSUE THREAD

**Issue by charliemoriarty** [a85/POSTMan-Chrome-Extension#6923]
**Describe the bug**
When making a GET request for an image which is being returned with no Content-Type header, the response is auto-detected as text in 7.3.2. The same URL is correctly detected as an image in 7.2.2, and in a Chrome browser.

**To Reproduce**
1. Make a request for an image URL which does not end in an extension, and does not contain a Content-Type header.
2. Click on Send
3. In the response box, Body shows as raw data instead of an image.

**Expected behavior**
An image shows in the response screen (this is the current behaviour in 7.2.2).

**Screenshots**
![Screenshot 2019-07-15 at 12 01 07](https://user-images.githubusercontent.com/33491122/61211904-b036ea80-a6f8-11e9-8156-6c0bb963b3e9.png)
![Screenshot 2019-07-15 at 12 02 50](https://user-images.githubusercontent.com/33491122/61211918-b4fb9e80-a6f8-11e9-94b9-61a96737fb0e.png)

**App information (please complete the following information):**
 - App Type: MacOS Desktop app
 - Postman Version: 7.3.2
 - OS: macOS Mojave 10.14.5

**Comment by shamasis** [comment]
Have you checked what clucking the “preview” button shows? The “pretty” and “raw” view should show text always.

**Comment by charliemoriarty** [comment]
@shamasis Yep, the 'preview' button is showing a similar thing:

![Screenshot 2019-07-16 at 11 08 30](https://user-images.githubusercontent.com/33491122/61286126-22bbcf00-a7ba-11e9-9816-1525d73cc04d.png)

**Comment by shamasis** [comment]
Ok. I think I am close to figuring out the cause here... can you send me the response headers that you get when you receive the response? @charliemoriarty ... and if this URL is public, having that instead will also get me what info I need to confirm the cause.

**Comment by saswatds** [comment]
Hi @charliemoriarty I have been able to reproduce this issue. In the `7.3.2` version of the app, we have made some changes to the response viewer to improve performance. During this refactoring, the auto-detection of content-type from the response-body got removed. Will fix this soon in a patch release.

Thanks for reporting this.

**Comment by charliemoriarty** [comment]
Thanks @saswatds and @shamasis ! The URL I was using was an internal one so I wasn't able to send it with the initial bug report, but I'm glad you were able to reproduce the issue. Thanks for looking into it 👍

**Comment by shamasis** [comment]
This should be now fixed and available in production. Can you try with latest app and let me know if the fix we intended, did actually fix your issue.

**Comment by charliemoriarty** [comment]
Thanks @shamasis! Unfortunately I'm still having the issue in Postman 7.5.0. I haven't had any luck finding a public version of the image URL I can send you, but the headers are still as follows, and the preview looks the same as the screenshot above:
```
Server: Microsoft-IIS/7.5
Date: Mon, 19 Aug 2019 08:17:43 GMT
Connection: close
Cache-Control: max-age=3600
Access-Control-Allow-Origin: *
```
Let me know if there's any more information you need!

**Comment by shamasis** [comment]
My bad. It’s scheduled in 7.6.0.

**Comment by codenirvana** [comment]
@charliemoriarty We've pushed a fix for this on `v7.6.0`. Can you update your app and confirm if this fixes your issue?

**Comment by charliemoriarty** [comment]
Thanks @codenirvana ! I can confirm that's fixed the issue in `v7.6.0` 🎉

**Comment by ahmafi** [comment]
For me, it was because I had the content type set to `image/jpg`, when I changed it to `image/jpeg` it worked.
